### ISSUE TITLE
Worried about Backups

### ISSUE THREAD

**Issue by mdrfkr** [electerious/Lychee#758]
I am at the get to know phase, so far I like what I see. I have installed the server on a Raspberry Pi to be serving my whole family. I have uploaded some pictures and everything works as supposed. BUT there are some things that troubles me. 
Some of the things that I dont love:
Whenever I modify data of a picture, it is not sotred as part of the metadata.
The names are changed.
The reason is that I needd backups, I want to backup my whole picture collecction, and not locally, if something happens to de drive I will lose everything. I may be able to script something to copy files to another commputer but I loose the names and all the modifications done to the tags etc.
Is there an easy way to backup the content outside the server?
Will there ever be?
I just need a good picture manager with over the web sharing cappablities and able to backup everything.

**Comment by chrobs** [comment]
Did you saw this info within the FAQ page?
[How can I backup my installation?](https://github.com/electerious/Lychee/blob/c0de7b8c63600e5a7001974bcb02c1c8e5bc3d99/docs/FAQ.md#how-can-i-backup-my-installation)

For a full backup it should be sufficient to copy the Lychee folder and a dump of the database to a different machine/media/etc

**Comment by mdrfkr** [comment]
thnks, no i didn't go through the FAQ. I will do it as proposed. Although I was hopping for something more non tech user friendly.

**Comment by chrobs** [comment]
Hi again,
I'm sorry, my first reply was a bit rough. I'm not able to say anything reliable to the metadata issue you mentioned. But I guess that if pictures' metadata is modified, lychee stores it in the database only and doesn't write it back to the pictures' exif information.

Concerning the backup request:
If you upload a photo, Lychee takes the md5 checksum of the current microtime as the new filename of the photo and stores it under the "uploads" directory structure on the file system. The association of a photo to an album (the structure you can see in the frontend) is managed within the database only. To backup the photos outside of lychee and retain the album structure should be possible but requires some scripting work.

Long story short: As far as I know, there is currently no non-tech way to do what you want.

**Comment by electerious** [comment]
This issue will be closed because I'm not actively working on Lychee anymore. The repository of Lychee will stay available, but the main development continues under the [Lychee Organisation](https://github.com/LycheeOrg). A fork maintained by the community. Important issues have been migrated. Feel free to re-add your issue(s) when necessary.
